lib/morandi/image_processor.rb

```diff
@@ -71,8 +71,9 @@ def write_to_png(fn, orientation=:any)
     pb.save(fn, 'png')
   end
 
-  def write_to_jpeg(fn, quality = 97)
```

Comment:
PR makes an implicit assumption that Morandi is primarily used 'for web consumption', but doesn't it also power the actual rendering process? If it does, do we have a confirmation from Geoff/clients that it's negligible for that purpose as well?

Reply:
Spoke with Geoff. He's explained it is used by the fulfillers as well so I will raise this back up to 97% and allow Bauhaus to override it.